packages/account/src/Duplicated/_common/base/client_base.js

```diff
@@ -0,0 +1,8 @@
+// TODO: [trader-remove-client-base] - Refactor Trader codebase to remove usage of this file
+
+// eslint-disable-next-line import/no-mutable-exports
+export let ClientBase;
+
+export const setClientBase = (client_base) => {
```

Comment:
Why is this file being duplicated here too? I can’t find the function `setClientBase` being used anywhere, could you perhaps link me to where it’s called?

Reply:
Ah good eye, sir. Thanks for reminding me, I forgot to initialize this once App.jsx is loaded. I've added the missing commands.